elastalert/alerters/email.py

```diff
@@ -112,7 +112,7 @@ def alert(self, matches):
             raise EAException("Error connecting to SMTP host: %s" % (e))
         except SMTPAuthenticationError as e:
             raise EAException("SMTP username/password rejected: %s" % (e))
-        self.smtp.sendmail(self.from_addr, to_addr, email_msg.as_string())
+        self.smtp.sendmail(self.from_addr, email_msg['To'].split(','), email_msg.as_string())
```

Comment:
On line 83 above, `email_msg['To']` is created by calling `join()` on `self.from_addr`. 

I might be missing something, but instead of calling `join()` and later calling `split()` on the same value, can we use `self.from_addr` directly?

`email_msg['To']` isn't manipulated in between the `join()` and the `split()` so it should be safe afaict.

Reply:
For whatever reason, the to_addr list is only sending the first recipient to my SMTP server, but doing the split on 'To' send all the configured addresses.

Reply:
Oh, I can't read. I'll try what you suggest.

Reply:
And I mistyped as well - I meant `self.to_addr`! I have updated my comment to reflect that.

Reply:
ERROR:elastalert:Uncaught exception running rule NI - User renamed: 'EmailAlerter' object has no attribute 'to_addr

Reply:
This PR would break CC/BCC deliveries. 

The local variable `to_addr` must contain the following when passed into `sendmail()`:
1.  the list of primary email recipients
2.  plus the list of cc recipients
3.  plus the list of bcc recipients

Note that I wrote _list_. I suspect the OP isn't aware that the `email:` parameter supports both a string and an array of strings. The correct way to specify multiple recipients is as follows:

```
email:
- someone@someplace.invalid
- anotherone@someotherplace.invalid
```

Reply:
@jertel  I was not aware of that! Maybe just something to clarify in the documentation for email alerts. Test and that works, thank you!